app/updates/BreakingNewsUpdate.scala

```diff
@@ -39,55 +38,66 @@ class BreakingNewsUpdate(val config: ApplicationConfiguration, val ws: WSClient,
     collection: ClientHydratedCollection,
     email: String
   ): Future[Result] = {
+    // TODO MRB:
+    // check we still log in the correct places
+    // does putting it in the web container set the "alert" flag?
```

Comment:
Had a little dig around to figure out why we need the alert set to true and there is no good reason for it but we need to work around the fronts publish model. Once we've alerted on a story and removed the story we need to click the remove button for the live version of the breaking news collections to be updated, clicking this remove will post to the same publish route as sending a breaking news alert but because the alert flag is set to false we don't send any alerts on draft articles.

Reply:
We agreed IRL to respect the flag for safety, whilst we're still unsure of what user actions would cause it to be set to false